communication/meeting-notes-archive/201808-201902_Community_Meeting_Minutes.md

```diff
@@ -2274,7 +2274,7 @@ Also @jorge for all the live stream work! You two rock
 *   [ 0:00 ] <strong>KEP o' the Week </strong>[Chris Hoge, @hogepodge, on behalf of Nishi Davidson, @d-nishi]
     *   Part of SIG Cloud Provider
         *   Coordinates stuff among all cloud providers
-    *   [https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0019-cloud-provider-documentation.md](https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0019-cloud-provider-documentation.md)  - Accepted 
+    *   [https://github.com/kubernetes/community/blob/master/sig-cloud-provider/CHARTER.md](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/CHARTER.md)  - Accepted 
```

Comment:
```suggestion
    *   [https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20180731-cloud-provider-docs.md](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20180731-cloud-provider-docs.md)  - Accepted 
```
KEPs were relocated to the [kubernetes/enhancements](https://github.com/kubernetes/enhancements/) repo late last year along with adopting a date-prefix. 👍 

If you could, please update both instances to reference the KEP link in the suggestion.